pyani_plus/private_cli.py

```diff
@@ -575,5 +575,94 @@ def log_anib(
     return 0
 
 
+@app.command()
+def log_dnadiff(
+    database: Annotated[str, typer.Option(help="Path to pyANI-plus SQLite3 database")],
+    # These are for the comparison table
+    query_fasta: Annotated[Path, typer.Option(help="Path to query FASTA file")],
+    subject_fasta: Annotated[Path, typer.Option(help="Path to subject FASTA file")],
+    mcoords: Annotated[
+        Path, typer.Option(help="Path to show-coords (.mcoords) output file")
+    ],
+    qdiff: Annotated[Path, typer.Option(help="Path to show-diff (.qdiff) output file")],
+    # Should we add --maxmatch (nucmer) and -m (deltafilter) parameters?
```

Comment:
We have ``maxmatch`` in the configuration table already, so a clear yes there I think.

I'm unclear what the other parameters are and if they should be available to the end user, or not.

Reply:
I think this might be with a discussion at some point. Both aniM and dnadiff methods use `nucmer` and `delta-filter`, but they use different parameters. 

For aniM we use `--mum` (`nucmer`) and `-1` (`delta-filter`), while for dnadiff we use`--maxmatch` (`nucmer`) and `-m` (`delta-filter`).

Reply:
These do not sound like user controllable parameters for ANI, but hard coded specifics of the methods.

Reply:
Saying all that... I'm not sure if we still want to leave the `--maxmatch` optional for `aniM` method.

Reply:
I had been meaning to ask why we had ``maxmatch`` in the database and if we would be using it or not...

Reply:
This is a question for @widdowquinn, I think 😄. I'm not sure when and why was `--maxmtach` introduced. Possible during the investigation of genome coverage being more than 1.0?